app/Models/V2/Projects/ProjectReport.php

```diff
@@ -439,11 +445,43 @@ public function getWorkdaysTotalAttribute(): int
         return $projectReportTotal + $sumTotals('paid') + $sumTotals('volunteer');
     }
 
+    public function getWorkdaysDirectTotalAttribute(): int
+    {
+        return $this->workdays_direct ?? 0;
+    }
+
+    public function getWorkdaysConvergenceTotalAttribute(): int
+    {
+        return $this->workdays_convergence ?? 0;
+    }
+
+    public function getNonTreeTotalAttribute(): int
+    {
+        if (empty($this->task_id)) {
+            return 0;
+        }
+
+        return TreeSpecies::where('speciesable_type', SiteReport::class)
+            ->whereIn('speciesable_id', $this->task->siteReports()->select('id'))
```

Comment:
Should this be including all site reports? Typically these types of queries are report status dependent. For instance `getWorkdaysTotalAttribute` in this class uses 
```
->whereIn('workdayable_id', $this->task->siteReports()->hasBeenSubmitted()->select('id'))
```